Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR.py

```diff
@@ -49,7 +68,7 @@ def air_acquire(self, hostname: str, profile: str, case_id: str, organization_id
             json_data=payload
         )
 
-    def air_isolate(self, hostname: str, organization_id: int, isolation: str) -> Dict[str, str]:
+    def air_isolate(self, hostname: str, organization_id: Optional[int], isolation: str) -> Dict[Any, str]:
```

Comment:
You have a type hint mismatch your return type is Dict[Any, str], but in line 148 the return type is Dict[Any, Any], please recheck your type hints.

Reply:
done